### PR TITLE
Update theme.css

### LINKED ISSUE #2860: Disable home page scrolling
### Environment
Latest Development branch

### What is the expected behaviour?
Scroll should be disabled

### What is the current behaviour?
Scroll is enabled

### How to reproduce the issue?
1. Login
2. Scroll is enabled in home page

### Screenshots or Screencast
https://www.loom.com/share/5a8fcb619b634c10b1ee0f6adc0e9510

### Any possible solutions?
Fix the height and disable scroll

### If the bug is confirmed, would you be willing to submit a PR?
No

### PR DESCRIPTION
**Pull request by AyushK-101**
Closes #2860

Kindly review @kavinvenkatachalam @withshubh

### CODE REVIEW

**Comment by gsmithun4** [comment]
@AyushK-101 Please sign CLA

**Comment by AyushK-101** [comment]
@gsmithun4 done

File: frontend/src/_styles/theme.scss [hunk 1]
```diff
@@ -6561,4 +6561,9 @@ tbody {
 }
 .jet-data-table td .textarea-dark-theme.text-container:focus {
   background-color: transparent !important; 
-}
\ No newline at end of file
+}
+
+.page-body{
+  height: calc(100vh - 2.5rem - 48px);
```
Comment (kavinvenkatachalam) [inline frontend/src/_styles/theme.scss:6567]:
```suggestion
  height: calc(100vh - 1.25rem - 48px);
```

**Review by kavinvenkatachalam — CHANGES_REQUESTED** [review]
@AyushK-101 Thanks for the PR.

There is some blank space at the bottom of the screen. Made a suggestion, please check and update the code.

image

**Review by kavinvenkatachalam — APPROVED** [review]
LGTM

**Comment by gsmithun4** [comment]
deployed https://tj-4538.herokuapp.com/

**Comment by withshubh** [comment]
Hey @AyushK-101  
Thanks for contributing to ToolJet. :raised_hands: 
Please submit the form to claim your Hacktoberfest swag: https://airtable.com/shr1qR6xhqsWO7LD7

**Pull request merged**